src/settings/login/LoginSettingsViewer.ts

```diff
@@ -218,19 +219,24 @@ export class LoginSettingsViewer implements UpdatableSettingsViewer {
 
 	private _renderActiveSessions(): Children {
 		return m(Table, {
-			columnHeading: ["client_label", "lastAccess_label", "IpAddress_label"],
-			columnWidths: [ColumnWidth.Largest, ColumnWidth.Small, ColumnWidth.Small],
+			columnHeading: ["client_label"],
+			columnWidths: [ColumnWidth.Largest],
 			showActionButtonColumn: true,
 			lines: this._sessions
 				.filter((session) => session.state === SessionState.SESSION_STATE_ACTIVE)
 				.map((session) => {
 					const thisSession = elementIdPart(locator.logins.getUserController().sessionId) === getElementId(session)
 
 					return {
-						cells: [
-							thisSession ? lang.get("thisClient_label") : session.clientIdentifier,
-							formatDateTimeFromYesterdayOn(session.lastAccessTime),
-							session.loginIpAddress ? session.loginIpAddress : "",
+						cells: () => [
+							{
+								main: thisSession ? lang.get("thisClient_label") : session.clientIdentifier,
+								info: [
+									`${lang.get("lastAccess_label")}: ${formatDateTimeFromYesterdayOn(session.lastAccessTime)}`,
```

Comment:
we should make this a key with a parameter

Reply:
Here's a screenshot in Hebrew (an RTL language):
![Screenshot from 2023-12-20 08-38-25(1)](https://github.com/tutao/tutanota/assets/48037381/9929de02-0a0e-450a-bb5f-63af18bb57c1)
It renders fine. If you like I can still change it to a key with a parameter if you can provide an example of this.

Reply:
I would also chime in that it should be a proper translation key

Reply:
> It renders fine. If you like I can still change it to a key with a parameter if you can provide an example of this.

`createActionStatus_msg` is one that has two parameters. It's used like this:

https://github.com/tutao/tutanota/blob/2939aa9f4356f0dc9f523ee5ce19d09e08ab979b/src/settings/ImportUsersViewer.ts#L141-L149

In this case, it would probably be something like 
"Last Access: {time}" and the translators can then do whatever, knowing that the `{time}` part will be replaced with a formatted datetime string.

Reply:
Thanks, I'll replace with a key now.